src/components/Causes/Causes.tsx

```diff
@@ -1,80 +1,146 @@
 import { Typography } from "@mui/material";
-import List from '@mui/material/List';
-import ListItem from '@mui/material/ListItem';
-import { textMargin } from "./Causes.styles";
-
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import { orderedListItemStyle, textMargin } from "./Causes.styles";
 
+const FootNote = ({ url, linkText }: { url: string; linkText: string }) => {
+  return (
+    <li style={orderedListItemStyle}>
+      <a href={url} rel="noopener noreferrer" target="_blank">
+        {linkText}
+      </a>
+    </li>
+  );
+};
```

Comment:
Moved this outside the Causes component so it doesn't get re-declared every render, also typed props.